android/src/main/java/com/razorpay/razorpay_flutter/RazorpayDelegate.java

```diff
@@ -176,4 +188,67 @@ public void onExternalWalletSelected(String walletName, PaymentData paymentData)
         sendReply(reply);
     }
 
+    public void setKeyID(String keyId,  Result result){
+        checkout.setKeyID(keyId);
+    }
+
+    public void linkNewUpiAccount(String customerMobile, String color, Result result){
+        Log.d("RazorpayDelegate" , "customerMobile "+customerMobile +" color "+color);
+        this.pendingResult = result;
+        Map<String, Object> reply = new HashMap<>();
+        checkout.upiTurbo.linkNewUpiAccount(customerMobile, color , new UpiTurboLinkAccountResultListener(){
+            @Override
+            public void onSuccess(@NonNull List<UpiAccount> upiAccounts) {
+                if(upiAccounts.isEmpty()){
+                    reply.put("data", "");
+                }else {
+                    reply.put("data", toJsonString(upiAccounts));
+                }
+                //sendReply(reply);
+            }
+
+            @Override
+            public void onError(@NonNull Error error) {
+              //  pendingResult.error(error.getErrorCode(), error.getErrorDescription(), toJsonString(error));
+            }
+        });
+    }
+
+
+    public void manageUpiAccounts(String customerMobile, String color, Result result){
+        this.pendingResult = result;
+        HashMap<Object, Object> reply = new HashMap<>();
+        checkout.upiTurbo.manageUpiAccounts(customerMobile, color , new GenericPluginCallback(){
+            @Override
+            public void onSuccess(@NonNull Object object) {
+            }
+
+            @Override
+            public void onError(@NonNull JSONObject jsonObject) {
+                pendingResult.error("", jsonObject.toString(), jsonObject.toString());
+            }
+        });
+    }
+
+    public  boolean isTurboPluginAvailable(Result result) {
+        this.pendingResult = result;
+
+        Map<String, Object> reply = new HashMap<>();
+        try {
+            Class.forName("com.razorpay.UpiTurboLinkAccountResultListener");
+            reply.put("isTurboPluginAvailable", true);
+            sendReply(reply);
+            return true;
+        } catch (ClassNotFoundException e) {
+            // Class not found, so it doesn't exist
+            reply.put("isTurboPluginAvailable", false);
+            sendReply(reply);
+            return false;
+        }
+    }
+
+    private String toJsonString(Object object){
+        return this.gson.toJson(object);
```

Comment:
When an upiAccount object is converted to json, does it also include the vpa object? @shibu000

Reply:
Yes @vivekshindhe